ui/apps/platform/src/Containers/Clusters/AddClusterPrompt.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {
+    Alert,
+    AlertVariant,
+    Button,
+    ButtonVariant,
+    Flex,
+    FlexItem,
+    Text,
+    TextContent,
+    TextVariants,
+} from '@patternfly/react-core';
+
+import SecurityFullColorPink from 'images/Security_FullColor_Pink.svg';
+
+function AddClusterPrompt() {
+    return (
+        <>
+            <Alert isInline variant={AlertVariant.success} title="You are ready to go!">
+                <p className="pf-u-font-weight-normal">
+                    You have successfully deployed a Red Hat Advanced Cluster Security platform. Now
+                    you can configure the clusters you want to secure.
+                </p>
+            </Alert>
+            <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                justifyContent={{ default: 'justifyContentCenter' }}
+                className="pf-u-text-align-center"
+                direction={{ default: 'column' }}
+            >
+                <FlexItem className="pf-u-mb-0">
+                    <img src={SecurityFullColorPink} alt="" style={{ height: '430px' }} />
+                </FlexItem>
+                <FlexItem className="pf-u-w-66">
+                    <TextContent className="pf-u-mb-xl">
+                        <Text component={TextVariants.h1} data-testid="congratulations">
+                            Configure the clusters you want to secure.
+                        </Text>
+                        <Text component={TextVariants.p} className="pf-u-font-weight-normal">
+                            Follow the instructions to add secured clusters for Central to monitor.
+                            <br />
+                            Upon successful installation, new clusters will be listed here.
+                        </Text>
+                    </TextContent>
+                </FlexItem>
+                <FlexItem>
+                    <Button
+                        variant={ButtonVariant.primary}
+                        component="a"
+                        target="_blank"
+                        rel="noopener noreferrer nofollow"
+                        href="https://docs.openshift.com/acs/3.72/installing/install-ocp-operator.html#adding-a-new-cluster-to-rhacs"
```

Comment:
Is this URL something we can determine dynamically, or do we just need to make a note to check this before each release?

Reply:
We plan to redirect to a page in the 3.73 doc set once we publish the 3.73 docs. So it's not dynamic. We may want a different approach in future.

Reply:
Why plan if redirects work already? https://github.com/stackrox/stackrox/pull/3799

Reply:
@msugakov The redirect that Lynne is referring to, is a 301 redirect of that doc link to a new improved documentation page about installation, which would not exist until 3.73 is released. The redirect in the documentation means we will not have to change this hard-coded link in the product itself.